crates/wasmprinter/src/lib.rs

```diff
@@ -15,6 +15,8 @@ use std::mem;
 use std::path::Path;
 use wasmparser::*;
 
+const MAX_LOCALS: u32 = 50000;
```

Comment:
Are you sure this is enough? While it might seem like it'd obviously be, I vaguely remember some preposterously large functions generated by Emscripten back in the asm.js days. Perhaps toolchains don't do those kinds of things anymore?

Reply:
I would copy whatever cranelift / wasmparser already have as their limit.

Reply:
Which is also `50000` :laughing:

Reply:
hah, fair enough 😁 I do wish the spec imposed limits here, but oh well.